### ISSUE TITLE
Stripped character '0' when alone in \lstinline

### ISSUE THREAD

**Issue by henrikt-ma** [brucemiller/LaTeXML#1651]
Here's a still broken variant of #1312:
```
\documentclass{article}
\usepackage{listings}
\begin{document}
Broken: \lstinline!0! (just one zero).

Works: \lstinline!00! (just two zeros).
\end{document}
```

**Comment by brucemiller** [comment]
Yikes, another one snuck through. fixed in 5b51eb05; thanks for the report!

**Comment by henrikt-ma** [comment]
Wow, that what quick!  Thanks!
